sdkproject/Assets/Mapbox/Unity/Map/RangeTileProvider.cs

```diff
@@ -19,11 +19,14 @@ public override void OnInitialized()
 			var centerTile = TileCover.CoordinateToTileId(_map.CenterLatitudeLongitude, _map.Zoom);
 			AddTile(new UnwrappedTileId(_map.Zoom, centerTile.X, centerTile.Y));
 
-			for (int x = (int)(centerTile.X - _west); x <= (centerTile.X + _east); x++)
+			for (int i= 0; i < 10000; i++)
```

Comment:
Did you intend to leave this here? I assume this was to force errors?

Reply:
Done!

Reply:
Good catch. Forgot to take that line out